src/stan/lang/rethrow_located.hpp

```diff
@@ -108,7 +108,7 @@ namespace stan {
       std::stringstream o;
       o << "Exception: " << e.what();
       if (line < 1) {
-        o << "  Found before start of program.";
+        o << "  in program.";
```

Comment:
Changing to as it was.  This message was confusing when the exception was due to bad variable declarations, but adding line numbers everywhere should solve that problem.